Source/Events/ReplicateItemListUpdate.lua

```diff
@@ -17,14 +18,20 @@ function Auctionator.Events.ReplicateItemListUpdate()
     local prices = {}
 
     for index = 0, C_AuctionHouse.GetNumReplicateItems() - 1 do
-      local _, _, count, _, _, _, _, _, _, buyoutPrice, _, _, _, _, _, _,
+      local name, _, count, _, _, _, _, _, _, buyoutPrice, _, _, _, _, _, _,
          itemId = C_AuctionHouse.GetReplicateItemInfo(index);
       local effectivePrice = buyoutPrice / count
+      local itemKey = tostring(itemId);
```

Comment:
Continuing the conversation from Discord, can we at least refactor this so we can `GetItemKeyFromReplicateResult` and `GetItemKeyFromWhatever`?  Would be nice to have item key generation in one place.

Reply:
In code below I see a `Auctionator.Utilities.ItemKeyFromLink`, for example...

Reply:
OK. ReplicateResults are stored in a global scope and accessed by index, so GetItemKeyFromReplicateResult will have a hidden global dependency. I can still make it though :)

Reply:
Done (w/out hidden dependencies)